### PR TITLE
Some small changes which also made the image smaller part deux

### PR DESCRIPTION
**Pull request by robbyoconnor**
I have made small changes and also managed to get the size down -- even with adding
dumb-init to the image! I have also merged the scripts to one script for both
backup and restore

I have also fixed the bug that caused cronbackups to not run due a small
oversight when I made mt changes since the backup.sh file is on the $PATH, oops.

### CODE REVIEW

File: cron-backup.sh [hunk 1]
```diff
@@ -3,15 +3,15 @@ set -e
...
 then
 echo "SCHEDULE environment not defined! Using 0 0 * * *"
 SCHEDULE="0 0 * * *"
 fi
 
-if [ ! -z "$DIRS" ] 
+if [ ! -z "$DIRS" ]
 then
-  crontab -l | { cat; echo "$SCHEDULE bash $PWD/backup.sh"; } | crontab -
+  crontab -l | { cat; echo "$SCHEDULE bash backup.sh backup"; } | crontab -
```
Comment (robbyoconnor) [inline cron-backup.sh]: This was the problem that caused things to fail..otherwise nothing here should ahve caused issues.

File: cron-backup.sh [hunk 2]
```diff
@@ -3,21 +3,18 @@ set -e
...
+if [ -z "$SCHEDULE" ]; then
+  echo "SCHEDULE environment not defined! Using 0 0 * * *"
+  SCHEDULE="0 0 * * *"
 fi
 
-if [ ! -z "$DIRS" ] 
-then
-  crontab -l | { cat; echo "$SCHEDULE bash $PWD/backup.sh"; } | crontab -
+if [ ! -z "$DIRS" ]; then
+  crontab -l | { cat; echo "$SCHEDULE bash backup.sh backup"; } | crontab -
```
Comment (robbyoconnor) [inline cron-backup.sh:12]: This was the bug  -- sorry about this. The script resides on the `$PATH` in `/usr/local/bin`

**Comment by HerbertYiga** [comment]
which ticket are you working on?Have you looked at this https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by robbyoconnor** [comment]
> which ticket are you working on?Have you looked at this https://wiki.openmrs.org/display/docs/Pull+Request+Tips

There is no ticket. I know how to make pull requests. I previously did work on this and forgot to change something in the cron-backup.sh file. and it broke...this was me fixing it and contributing back.

**Comment by robbyoconnor** [comment]
> which ticket are you working on?Have you looked at this https://wiki.openmrs.org/display/docs/Pull+Request+Tips

@HerbertYiga, 
1. There is no ticket. 
2. I know how to make pull requests. I previously did work on this and forgot to change something in the cron-backup.sh file. and it broke...this was me fixing it and contributing back.

**Comment by robbyoconnor** [comment]
> which ticket are you working on?Have you looked at this https://wiki.openmrs.org/display/docs/Pull+Request+Tips

@HerbertYiga, 
1. There is no ticket. 
2. I know how to make pull requests. If you look at the tags you'll notice I'm a member of the OpenMRS GitHub Organization. 
3. I previously did work on this and forgot to change something in the cron-backup.sh file. and it broke...this was me fixing it and contributing back.

**Comment by cintiadr** [comment]
Yeah, @HerbertYiga this script usually don't work for infrastructure repositories. 

Mostly people already talked to me. 

What you can do instead is making sure I'm always added as a reviewer.

**Pull request closed without merging**